css/properties/-webkit-user-drag.json

```
@@ -20,7 +20,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤5.1"
```

Already present in Safari 4.0 (https://github.com/caugner/css-properties/commit/33ce1ea279967afb2272f8b51aa53a147afcb375).

```suggestion
              "version_added": "≤4"
```